New pages: HTMLInputElement step, name, and defaultValue properties (#35614)
mdn/content: files/en-us/web/api/htmlinputelement/defaultvalue/index.md

---
title: "HTMLInputElement: defaultValue property"
short-title: defaultValue
slug: Web/API/HTMLInputElement/defaultValue
page-type: web-api-instance-property
browser-compat: api.HTMLInputElement.defaultValue
---

{{ApiRef("HTML DOM")}}

> The **`defaultValue`** property of the {{domxref("HTMLInputElement")}} interface indicates the original (or default) value of the {{HTMLElement("input")}} element's [`value`](/en-US/docs/Web/HTML/Element/input#value) attribute as originally specified in the HTML that created the control.

Reviewer: Our standard lingo is "it reflects the `<input>` element's `value` attribute". No need to say "as originally specified...".

```suggestion
The **`defaultValue`** property of the {{domxref("HTMLInputElement")}} interface indicates the original (or default) value of the {{HTMLElement("input")}} element. It reflects the element's [`value`](/en-US/docs/Web/HTML/Element/input#value) attribute.
```

## Value

A string representing the default, or original, value of the `<input>` element.

## Example

> This example demonstrates retrieving the original and current values of an input element.

Reviewer: TBH, I don't think this example is interesting enough to warrant a live sample, and I'd very much prefer it to be two lines like
```js
const inputElement = document.getElementById("planet");
console.log(inputElement.defaultValue); // "Azarath"
```
Just my preference though.
